src/app/window-handler.ts

```diff
@@ -431,6 +431,49 @@ export class WindowHandler {
         return browserWindow && window === browserWindow;
     }
 
+    /**
+     * Move window to the same screen as main window
+     */
+    public moveWindow(windowToMove: BrowserWindow) {
+        const allWindows = BrowserWindow.getAllWindows();
+
+        const parentWindow = allWindows.find((window) => {
+            return (window as ICustomBrowserWindow).winName === 'main';
+        });
+
+        if (parentWindow && windowExists(parentWindow)) {
+            const display = electron.screen.getDisplayNearestPoint({x: parentWindow.getBounds().x, y: parentWindow.getBounds().y});
+
+            logger.info('window-handler: moveWindow, display: ' + JSON.stringify(display.workArea));
+            logger.info('window-handler: moveWindow, windowToMove: ' + JSON.stringify(windowToMove.getBounds()));
+
+            if (display.workArea.width < windowToMove.getBounds().width) {
+                windowToMove.setSize(display.workArea.width, windowToMove.getBounds().height);
+            }
+
+            if (display.workArea.height < windowToMove.getBounds().height) {
+                windowToMove.setSize(windowToMove.getBounds().width, display.workArea.height);
+            }
+
+            let positionX = Math.trunc(display.workArea.x + display.workArea.width / 2 - windowToMove.getBounds().width / 2);
+            if (positionX < display.workArea.x) {
+                positionX = display.workArea.x;
+            }
+
+            let positionY = Math.trunc(display.workArea.y + display.workArea.height / 2 - windowToMove.getBounds().height / 2);
+            if (positionY < display.workArea.y) {
+                positionY = display.workArea.y;
+            }
+
+            logger.info('window-handler: moveWindow, positionX: ' + positionX);
+            logger.info('window-handler: moveWindow, positionY: ' + positionY);
+
+            windowToMove.setPosition(positionX, positionY);
+            // Because of a bug for windows10 we need to call setPosition twice
```

Comment:
@johankwarnmarksymphony Can you please add the link to the issue so, it will be easier to track